x-pack/plugins/canvas/shareable_runtime/supported_renderers.js

```diff
@@ -40,4 +38,7 @@ export const renderFunctions = [
   text,
 ];
 
+export const addSupportedRenderFunctions = (renderers = []) =>
```

Comment:
nit: honestly I'm not a fan modify a global(module) variable using that way.  for some cases it can we a reason of issues that are difficult to debug.

As an simple example:
```
let a = [1, 2, 3];
const b = a;
a = [...a, 4];

console.log(a, b);
```
I do not know how we will use it in the future, but if now it is not a constant value, then it should be processed in the setup hook

Reply:
How do you want to process it in the setup hook? I was thinking about making this variable private and represent getter for it. It would prevent from the wrong usage.

Reply:
Fixed.